Issue running lerna bootstrap

I've followed the desktop build instructions laid out in the [README](https://github.com/TryQuiet/quiet/blob/develop/packages/desktop/README.md), and in the `npm run lerna bootstrap` step, bootstrap fails while building the backend package with the error:

```
lerna ERR! lifecycle "prepare" errored in "@quiet/backend", exiting 1
```


I ran `npm run prepare` in packages/backend to try and get a more useful error message, and webpack completes but reports that it can't find some modules. For example:
```
ERROR in ./src/nest/auth/services/crypto/crypto.service.ts 4:0-46
Module not found: Error: Can't resolve '@localfirst/auth' in '/Users/adamdefelice/src/quiet/packages/backend/src/nest/auth/services/crypto'
 @ ./src/nest/auth/sigchain.ts 7:0-65 56:23-41
 @ ./src/nest/auth/sigchain.service.ts 12:0-38 75:25-40 81:25-38
 @ ./src/nest/connections-manager/connections-manager.service.ts 48:0-59 802:8-23
 @ ./src/backendManager.ts 7:0-99 50:39-64 94:43-68 98:43-68
```
Not sure if this is a red herring.

I replicated this issue cloning the repo from scratch on a friend's machine. Both builds were on mac os.
Any help is appreciated :)

--- a/.github/workflows/e2e-android-self.yml
+++ b/.github/workflows/e2e-android-self.yml
@@ -21,7 +21,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm i
-          npm run lerna bootstrap --scope @quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle
+          npm run bootstrap --scope @quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle
 
       - name: Pull binaries
         run: |

--- a/.github/workflows/e2e-android.yml
+++ b/.github/workflows/e2e-android.yml
@@ -20,7 +20,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm ci
-          npm run lerna bootstrap -- --scope=\'{@quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle}\'
+          npm run bootstrap -- --scope=\'{@quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle}\'
 
       - name: Pull binaries
         run: |

--- a/.github/workflows/e2e-ios-self.yml
+++ b/.github/workflows/e2e-ios-self.yml
@@ -22,7 +22,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm i
-          npm run lerna bootstrap --scope @quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle
+          npm run bootstrap --scope @quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle
 
       - name: Install pods
         run: |

--- a/.github/workflows/e2e-ios.yml
+++ b/.github/workflows/e2e-ios.yml
@@ -20,7 +20,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm ci
-          npm run lerna bootstrap -- --scope=\'{@quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle}\'
+          npm run bootstrap -- --scope=\'{@quiet/eslint-config,@quiet/logger,@quiet/common,@quiet/types,@quiet/state-manager,@quiet/backend,@quiet/identity,@quiet/mobile,backend-bundle}\'
 
       - name: Pull binaries
         run: |

--- a/packages/desktop/README.md
+++ b/packages/desktop/README.md
@@ -3,23 +3,40 @@
 Running the desktop version of Quiet should be straightforward on Mac and Linux. On Windows we recommend using git-bash or just wsl.
 Here are the steps:
 
-0. Install `patch` (e.g. via your Linux package manager)
-1. Use `Node 18.12.1` and `npm 8.19.2`. We recommend [nvm](https://github.com/nvm-sh/nvm) for easily switching Node versions, and if this README gets out of date you can see the actual version used by CI [here](https://github.com/TryQuiet/quiet/blob/master/.github/actions/setup-env/action.yml). If you are using nvm, you can run `nvm use` in the project's root to switch to the correct version.
-2. In `quiet/` (project's root) install monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
+1. Install `patch` (e.g. via your Linux package manager)
+
+2. Use `Node 18.12.1` and `npm 8.19.2`. We recommend [nvm](https://github.com/nvm-sh/nvm) for easily switching Node versions, and if this README gets out of date you can see the actual version used by CI [here](https://github.com/TryQuiet/quiet/blob/master/.github/actions/setup-env/action.yml). If you are using nvm, you can run `nvm use` in the project's root to switch to the correct version.
+
+3. Initialize submodules in the project's root:
+
+```
+git submodule update --init --recursive
+```
+
+4. Install python setuptools
+
+```
+python3 -m pip install setuptools
+```
+
+5. In `quiet/` (project's root) install monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
 
 ```
 npm i lerna@6.6.2
 npm i typescript@4.9.3
+npm i -g pnpm@9.12.1 // may be needed depending on configuration
+npm install
 npm run bootstrap
 ```
 
-If you run into problems please double check if you have exact version Node and NPM as listed in point 0.
+If you run into problems please double check if you have exact version Node and NPM as listed in point 1.
 
-3. In `quiet/packages/desktop` run:
+6. In project root run,
 
 ```
-npm run start
+npm run start:desktop
 ```
+
 ----
 
 ## Versioning packages

--- a/packages/desktop/README.md
+++ b/packages/desktop/README.md
@@ -13,11 +13,7 @@ Here are the steps:
 git submodule update --init --recursive
 ```
 
-4. Install python setuptools
-
-```
-python3 -m pip install setuptools
-```
+4. Install python3 and setuptools through your preferred method. (used by node-gyp)
 
 5. In `quiet/` (project's root) install monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
 

--- a/packages/desktop/README.md
+++ b/packages/desktop/README.md
@@ -10,7 +10,7 @@ Here are the steps:
 ```
 npm i lerna@6.6.2
 npm i typescript@4.9.3
-npm run lerna bootstrap
+npm run bootstrap
 ```
 
 If you run into problems please double check if you have exact version Node and NPM as listed in point 0.

--- a/packages/e2e-tests/README.md
+++ b/packages/e2e-tests/README.md
@@ -3,7 +3,7 @@
 *  Install chromedriver correctly before running the tests:
 
 `export ELECTRON_CUSTOM_VERSION=23.0.0`
-`npm run lerna bootstrap`
+`npm run bootstrap`
 
 
 *  Run jest:
@@ -18,7 +18,7 @@ To run against binaries built locally (which will be in the `/dist` folder) you 
 npm run test:localBinary
 ```
 
-This passes the `IS_LOCAL` flag and will use local binaries in the `/dist` directory (if the OS has been configured for it in the tests).  Check the README in the `desktop` package for information on building binaries for each OS.  
+This passes the `IS_LOCAL` flag and will use local binaries in the `/dist` directory (if the OS has been configured for it in the tests).  Check the README in the `desktop` package for information on building binaries for each OS.
 
 *You must compile the binary prior to running this command or it will fail!*
 

--- a/packages/mobile/README.md
+++ b/packages/mobile/README.md
@@ -12,7 +12,7 @@ Quiet Mobile is a React Native app for Android and iOS that shares a Node.js [ba
 
     ```bash
     npm install
-    npm run lerna bootstrap
+    npm run bootstrap
     ```
 
 1. On your host, install [adb](https://developer.android.com/studio/command-line/adb) (Android Debug Bridge) to communicate with your Android device.

--- a/packages/mobile/README.md
+++ b/packages/mobile/README.md
@@ -8,6 +8,8 @@ Quiet Mobile is a React Native app for Android and iOS that shares a Node.js [ba
 
 1. If not on Mac (which comes preinstalled with `patch`), install `patch` (e.g. via your Linux package manager).
 
+1. Install python3 and setuptools through your preferred method. (used by node-gyp)
+
 1. In the root directory of `quiet/`, install the monorepo's dependencies and bootstrap the project with lerna. It will take care of the package's dependencies and trigger a prepublish script which builds them.
 
     ```bash

